.github/workflows/tact.yml

```diff
@@ -57,14 +57,14 @@ jobs:
       - name: Compare Tact version from CLI flag `--version` against package.json
         if: runner.os != 'Windows'
         run: |
-          if [ "$(./bin/tact --version)" != "$(jq -r '.version' < package.json)" ];
+          if [ "$(./bin/tact.mjs --version)" != "$(jq -r '.version' < package.json)" ];
           then false
           fi
 
       - name: Tact CLI non-zero exit code
         if: runner.os != 'Windows'
         run: |
-          ! ./bin/tact --config test/tact-cli/tact.config.json
+          ! ./bin/tact.mjs --config test/tact-cli/tact.config.json
```

Comment:
This should still be called as `tact`, not `tact.mjs`

Reply:
Does it? npm performs the aliasing to `tact` only when we install the package from the registry. But the local use only has the present files, so `tact.mjs` it is

Reply:
Why not then install it locally from the repo and perform all the CLI tests using the installed version?

Reply:
Good point!